bin/chrome_launcher.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Download Extension
+extension_setup() {
+  ps_analysis_tool_version=v0.3.1
+  cd /tmp/
```

Comment:
When we do `cd /tmp/` the directory is changed to that, and does not revert back to where we started, can we please ensure that after downloading and setting up the extension, we go back to the directory where we were?

Reply:
@MiteshShah 
Can this be handled?